scripts/setup/config.emacs

```diff
@@ -0,0 +1,13 @@
+(add-to-list 'load-path (substitute-in-file-name "${HOLBA_OPT_DIR}/hol_k12_holba/tools"))
+
+(cua-mode)
+
+(load "hol-mode.el")
+;(load "hol-unicode.el")
```

Comment:
If this mode/script/whatever automatically translates ASCII operators in the equivalent Unicode ones, you could add a comment saying that this is against HolBA coding style.

Reply:
That's a good point. I'll check later.